npm/private/npm_translate_lock.bzl

```diff
@@ -135,12 +135,28 @@ _ATTRS = {
         doc = """If true, runs preinstall, install and postinstall lifecycle hooks on npm packages if they exist""",
         default = True,
     ),
+    "lifecycle_hooks_envs": attr.string_list_dict(
+        doc = """Environment variables applied to the preinstall, install and postinstall lifecycle hooks on npm packages.
+            Such as some package can define a PREBUILT_BINARY environment variable to download a os specific binary file
```

Comment:
I think delete this sentence (this line + next), the example shows it.

Reply:
Got it , deleted.